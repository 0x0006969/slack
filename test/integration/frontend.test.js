const request = require('supertest');
const nock = require('nock');

const helper = require('.');

const { probot } = helper;

const fixtures = require('../fixtures');

const access = fixtures.slack.oauth.token;

describe('Integration: frontend', () => {
  test('/', async () => {
    const res = await request(probot.server).get('/');

    expect(res.status).toBe(200);
    expect(res.text).toMatchSnapshot();
  });

  test('/denied', async () => {
    const res = await request(probot.server).get('/denied');

    expect(res.status).toBe(200);
    expect(res.text).toMatchSnapshot();
  });
});
