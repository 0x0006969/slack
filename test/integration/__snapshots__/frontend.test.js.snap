// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Integration: frontend / 1`] = `[MockFunction cache.get]`;

exports[`Integration: frontend / 2`] = `[MockFunction cache.set]`;

exports[`Integration: frontend / 3`] = `[MockFunction cache.fetch]`;
